src/asm/symbol.c

```diff
@@ -538,7 +538,7 @@ struct Symbol *sym_AddLocalLabel(char const *name)
 		 * as long, and then that this was the entirety of the `Parent` part of `name`.
 		 */
 		if (labelScope[i] != '\0' || name[i] != '.')
-			error("Not currently in the scope of '%.*s'\n", parentLen, name);
+			error("Not currently in the scope of '%.*s'\n", (unsigned int)parentLen, name);
```

Comment:
As far as I'm aware, the size field for `.*` is supposed to be `int`? (Yes, that's absolutely bonkers, but that's how it is)

Reply:
Wow, you are right! That is bonkers.